includes/polling/functions.inc.php

```diff
@@ -45,10 +45,10 @@ function poll_sensor($device, $class, $unit)
 
             if (file_exists('includes/polling/sensors/'. $class .'/'. $device['os'] .'.inc.php')) {
                 require_once 'includes/polling/sensors/'. $class .'/'. $device['os'] .'.inc.php';
+            } else {
+                $sensor_value = trim(str_replace('"', '', $snmp_data[$sensor['sensor_oid']]));
```

Comment:
We shouldn't need to move that to here, it can be left generically below as no sensor value should have  a " in it.